app/elements/editor-console/editor-console.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Created by Paul on 7/20/2015.
+ */
+Polymer({
+  is: 'editor-console',
+  properties:{
+    editor: {
+      type: Object,
+      reflectToAttribute: true,
+      notify: true
+    }
+  },
+  observers: [
+    'afterPropertiesSet(editor)'
+  ],
+  afterPropertiesSet: function(){
+    this.setLanguage();
+  },
+  ready: function(){
+    this.$.console.textarea.value = 'Console';
+//       setTimeout(this.setLanguage(), 1);
+  },
+  runCode: function(){
+    this.$.ajaxRequest.method='POST';
+    this.$.ajaxRequest.body = JSON.stringify({'code': this.getCode(), 'language': this.getSelectedLanguage()});
+    this.$.ajaxRequest.url='http://localhost:8080/testRoute';
```

Comment:
Hardcoded reference to localhost. Use something like `window.location` to derive the correct endpoint.